components/ai_chat/core/common/mojom/ai_chat.mojom

```diff
@@ -118,6 +118,8 @@ struct SearchStatusEvent {
 
 struct CompletionEvent {
   string completion;
+  // Whether the completion was generated from Brave Search SERP.
+  bool from_brave_search_SERP = false;
```

Comment:
What does the default value actually do? Judging by the changes, we still have to set this to false most places we use it?

Reply:
On the top of my head, it could be used when you do `CompletionEvent event` or `CompletionEvent::New()`.

Reply:
Ah okay, so only when we're not passing in arguments to `::New()`? That makes sense, thanks @yrliou